src/components/Footer.astro

```diff
@@ -34,7 +34,7 @@ import { donationOptions } from '~data';
 			{
 				donationOptions.length > 0 && (
 					<Fragment>
-						<p>
+						<p style="margin-block-start: 1em;">
```

Comment:
Is there a reason we're using `style` attributes here and not classes?

Reply:
Yes: laziness. I've done this in a few small places and I want to come back and tidy up.